### PR TITLE
Feature/#54 fixes errors

### PR DESCRIPTION
**Pull request by flxtagi**
your fixes included (branch #95 Refactor)

fixes for: 
- undefined lenght when comming back from a client-rendered site. 
- manifest. 
- some libraries updated. should fix deprecation warining
- favicon fix
- no /static all /public folder
- hydration error while skeleton loading for posts
- postList fixes if undefined. 

TBD: discuss if this router.replace is a good enough solution

### CODE REVIEW

File: src/components/post/PostList.tsx
Comment (rudigier) [inline src/components/post/PostList.tsx:1]: Please revert changes. Use instead Math.rand -> i % 2

File: src/components/helpers/PostSkeleton.tsx
Comment (rudigier) [inline src/components/helpers/PostSkeleton.tsx:1]: please revert changes

File: src/pages/_app.tsx
Comment (rudigier) [inline src/pages/_app.tsx:1]: please remove /public/

File: src/pages/index.tsx [hunk 1]
```diff
@@ -13,10 +14,23 @@ export default function PageHome({
...
 	}
 
+	// if posts are undefined we are re-routing (replacing!) to the same page to trigger the getServerSideProps
+	// this happens rarely but we need to handle it, when a user navigates back from a client-side rendered page
+	// and the page is not in the cache anymore or next sends just json data to the client instead of the full page
+	if (!posts) {
+		router.replace(router.asPath);
+	}
+
+	const canLoadMore = postCount > posts?.length || false;
```
Comment (rudigier) [inline src/pages/index.tsx:32]: please revert this line

**Review by rudigier — CHANGES_REQUESTED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (flxtagi) [reply src/pages/index.tsx]: changed

**Review by rudigier — APPROVED** [review]
ok for me

**Pull request merged**